test/runtests.jl

```diff
@@ -129,6 +130,10 @@ include("subdir/file3.jl")
 include(joinpath(@__DIR__, "subdir", "file4.jl"))
 otherfile = "file5.jl"
 include(otherfile)
+macro mac$(fbase)()
```

Comment:
Let's start with the basics: the point of the test is to exercise a macro? I spent some time trying to figure out whether this was intended to be OSX-specific, since we've had trouble with OSX. ("mac" = "macro" and not "macintosh"). OK, the purpose is much clearer now. 

Since we can't control hash order, and in any case you've fixed that now anyway, I'm fine with a test that might have passed before but *will* pass now.